### DISCUSSION TITLE
VS code Different Layout

### DISCUSSION THREAD

**Discussion by DeutschZaw** [giswqs/geog-312#91] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/53793550-bec3-40ac-b133-70010a17cdfb)
On the instructor's screen, there are tabs (branches, commit etc) as shown in figure. but on my side, there are only three : source control, source control graph and source control repositories. but when I click 3 dots butoon it show tabs like commit , slashes etc (see in 3rd pic). but it is not the same as you. 

Though it is not a bit deal, what should I do to make it the same as on your VS code layout?
![layout](https://github.com/user-attachments/assets/eba2a46d-d0fe-43e2-81e1-e24c4b4ef023)

**Comment by giswqs** [comment, marked as answer]
Click on the three dots next to the repository to see if those options are checked. 

![image](https://github.com/user-attachments/assets/ce2040d7-4e5a-498b-885a-03c043fd36ac)

**Comment by DeutschZaw** [comment]
![image](https://github.com/user-attachments/assets/3cdc00b2-bcaf-4cef-a45b-38835ea2fddf)
I have already tried it, sir. Found only 3 folder as in photo.
